pkgs/applications/networking/instant-messengers/gurk-rs/default.nix

```diff
@@ -23,12 +23,7 @@ rustPlatform.buildRustPackage rec {
 
   cargoLock = {
     lockFile = ./Cargo.lock;
-    outputHashes = {
-      "curve25519-dalek-3.2.1" = "sha256-T/NGZddFQWq32eRu6FYfgdPqU8Y4Shi1NpMaX4GeQ54=";
-      "libsignal-protocol-0.1.0" = "sha256-gapAurbs/BdsfPlVvWWF7Ai1nXZcxCW8qc5gQdbnthM=";
-      "libsignal-service-0.1.0" = "sha256-CrfTdUcxP591pigS2069gEjzy5jSRz7mHORLCodQDSE=";
-      "presage-0.3.0" = "sha256-Ptyjf5/SI8ftjiIxK+gVya5Cmv5sOBmWXM8ZveVV7Pc=";
-    };
+    allowBuiltinFetchGit = true;
```

Comment:
I don't think this is allowed in nixpkgs. You can try [nix-update](https://github.com/Mic92/nix-update) if you find updating these hashes cumbersome.

Reply:
Hmm, thank you. I guess I misunderstood this [section](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md#importing-a-cargolock-file-importing-a-cargolock-file).

Reply:
I'm testing `nix-update` right now. Wow, @figsoda, it's seriously amazing, thank you! :smile: :beers:

Just running the build now, and so far it's looking really good.